src/control/lib/control/system.go

```diff
@@ -248,7 +248,7 @@ func (fwdr *EventForwarder) OnEvent(ctx context.Context, evt *events.RASEvent) {
 
 	req := &SystemNotifyReq{
 		Sequence: <-fwdr.seq,
-		Event:    evt,
+		Event:    evt.WithIsForwarded(true),
```

Comment:
I'm not sure how this works... Isn't this helper setting an unexported field? If so, the value won't survive the trip across gRPC, unless I'm missing something.

Reply:
The protobuf definition will need to be updated as well, won't it?

Reply:
fixed implementation in https://github.com/daos-stack/daos/pull/4435 , forwarded only exists in the native type and gets set on receipt in the gRPC ClusterEvent handler https://github.com/daos-stack/daos/pull/4435/files#diff-082855a22f45199564e850828dfc080b0d4c1b0dfa128256011737ccff1c1123R744 https://github.com/daos-stack/daos/pull/4435/files#diff-6e63051d5e46ac18fba5b9ba7508beda791d44a7dda83cebf0b2a3b95764eacbR299 , hence why it's un-exported